src/Frontend/Themes/Fork/Core/Layout/Sass/screen.scss

```diff
@@ -1,4 +1,4 @@
-@charset 'UTF-8';
+@charset "UTF-8";
```

Comment:
why some things with double quote and some things with single quote?

Reply:
My IDE told me to :) Some context: https://github.com/KittyGiraudel/sass-guidelines/issues/255

Apparently double quotes is the standard way to use @charset (TIL).